packages/valory/skills/market_manager_abci/tests/test_payloads.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2024 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+"""This module contains the transaction payloads for the market manager abci."""
+
+from packages.valory.skills.market_manager_abci.payloads import UpdateBetsPayload
+
+
+def test_check_stop_trading_abci_payload() -> None:
+    """Test `UpdateBetsPayload`."""
+
+    payload = UpdateBetsPayload(sender="sender", bets_hash="dummy bets hash")
+
+    assert payload.bets_hash == "dummy bets hash"
+    assert UpdateBetsPayload.from_json(payload.json) == payload
```

Comment:
why is this testing UpdateBetsPayload?

Reply:
It's the only payload of this abci.

Reply:
test seems missplaced

Reply:
Maybe the method's name is incorrect instead @annasambrook.

Reply:
yep, it was incorrect, should be fixed now